{
  "cveId": "CVE-2019-25163",
  "eventName": "New CVE Received",
  "cveChangeId": "2D3F67B7-509F-4484-842A-4D698CC4E501",
  "sourceIdentifier": "416baaa9-dc9f-4396-8d5f-8c081fb06d67",
  "created": "2025-09-16T15:15:42.377",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "Rejected reason: ** REJECT ** DO NOT USE THIS CVE RECORD. ConsultIDs: none. Reason: This record was in a CNA pool that was not assigned to any issues during 2019. Notes: none."
    }
  ]
}